### ISSUE TITLE
Package specified in interface or trait PHPdoc leads to false positive

### ISSUE THREAD

**Issue by MHajoha** [moodlehq/moodle-local_moodlecheck#96]
**Repro**
1. Create a PHP file containing only an interface or a trait, and specify the package in the PHPdoc of that artifact.
2. Add a function to the artifact, omitting the `@package` tag from its PHPdoc
3. Run moodlecheck on the file

<details>
<summary>Example file</summary>

```php
<?php
// This file is part of Moodle - https://moodle.org/
//
// Moodle is free software: you can redistribute it and/or modify
// it under the terms of the GNU General Public License as published by
// the Free Software Foundation, either version 3 of the License, or
// (at your option) any later version.
//
// Moodle is distributed in the hope that it will be useful,
// but WITHOUT ANY WARRANTY; without even the implied warranty of
// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
// GNU General Public License for more details.
//
// You should have received a copy of the GNU General Public License
// along with Moodle.  If not, see <http://www.gnu.org/licenses/>.

namespace qtype_questionpy\form;

/**
 * Description.
 *
 * @package    qtype_questionpy
 */
trait repro {

    /**
     * Description.
     *
     * @return void
     */
    function def() {}
}
```

</details>

**Expected Behaviour**
The missing `@package` tag is not flagged since the one on the artifact suffices.

**Actual Behaviour**
> Line 29: Package is not specified for function def. It is also not specified in file-level phpdocs

**Comment by stronk7** [comment]
Awesome @MHajoha, thanks! Looking to #97 soon!
